docs/shared/packages/powerpack-owners/powerpack-owners-plugin.md

```diff
@@ -1,6 +1,326 @@
+
+{% callout title="This plugin requires an active Nx Powerpack license" %}
+In order to use `@nx/powerpack-owners`, you need to have an active Powerpack license. If you don't have a license or it has expired, the syncing process will stop working and you'll need to manually maintain your CODEOWNERS file.
+{% /callout %}
+
+## Setup
+
+1. [Activate Powerpack](/recipes/installation/activate-powerpack) if you haven't already
+2. Install the package
+
+   ```shell
+   nx add @nx/powerpack-owners
+   ```
+
+3. Configure Ownership
+
+   Configure the `@nx/powerpack-owners` plugin in the `nx.json` file or in individual project configuration files. Consult the [Owners Configuration Reference](#owners-configuration-reference) section for more details.
+
+4. Configure the [Sync Generator](/concepts/sync-generators) and CI
+
+Make sure that the `@nx/powerpack-owners:sync-codeowners-file` generators is registered as a `globalGenerator` in `nx.json`:
+
+```jsonc {% fileName="nx.json" %}
+{
+  "sync": {
+    "globalGenerators": ["@nx/powerpack-owners:sync-codeowners-file"]
+  }
+}
+```
+
+Add `nx sync:check` to the beginning of the CI process.
+
+```yaml
+- name: Ensure the workspace configuration is in sync
+  run: npx nx sync:check
+```
+
+It is also often helpful to add `nx sync` as a git push hook or git commit hook.
+
+## Owners Configuration Reference
+
+{% tabs %}
+{% tab label="GitHub" %}
+
+```jsonc {% fileName="nx.json" %}
+{
+  // Can be set to true instead of an object to accept all defaults
+  "owners": {
+    // Options are `github`, `bitbucket` or `gitlab`. (Optional) Defaults to `github`
+    "format": "github",
+    // (Optional) Default changes based on format: `.github/CODEOWNERS`, `.bitbucket/CODEOWNERS`, `.gitlab/CODEOWNERS`
+    "outputPath": "CODEOWNERS",
+    // (Optional)
+    "patterns": [
+      {
+        "description": "A description of the rule",
+        "owners": ["@joelovesrust"],
+        // specify either projects or files, not both
+        // Can be any project specifier that could be used in `nx run-many`
+        // See https://nx.dev/nx-api/nx/documents/run-many
+        "projects": ["my-rust-app", "rust-*", "tag:rust"],
+        // File globs
+        "files": [".github/workflows/**/*"]
+      }
+    ]
+  }
+}
+```
+
+{% /tab %}
+{% tab label="Bitbucket" %}
+
+```jsonc {% fileName="nx.json" %}
+{
+  // Can be set to true instead of an object to accept all defaults
+  "owners": {
+    // Options are `github`, `bitbucket` or `gitlab`. (Optional) Defaults to `github`
+    "format": "bitbucket",
+    // (Optional) Default changes based on format: `.github/CODEOWNERS`, `.bitbucket/CODEOWNERS`, `.gitlab/CODEOWNERS`
+    "outputPath": "CODEOWNERS",
+    // (Optional)
+    "patterns": [
```

Comment:
This shouldn't be optional for bitbucket, only for gitlab where sections is supported. Can you verify this, it's a bug if so